src/explore-education-statistics-admin/src/components/PageErrorBoundary.tsx

```diff
@@ -19,6 +19,16 @@ interface State {
 class PageErrorBoundary extends Component<RouteComponentProps, State> {
   public state: State = {};
 
+  private unregisterCallback?: () => void;
+
+  private errorPages = {
+    forbidden: () => {
+      this.setState({
+        errorCode: 403,
+      });
+    },
+  };
+
```

Comment:
Has this moved for a reason? Not that I think it's an issue if there is no reason - just interested.